duckduckgo/duckduckgo-privacy-extension
# Update email protection in settings

**Asana:** https://app.asana.com/0/312629933896096/1200412250706923/f
**Reviewer:** @jonathanKingston 

## Description:
Changes Email Protection text in the settings and adds a visible Autofill Disabled state (in previous version, when autofill was disabled we just didn't show it).

## Steps to test this PR:
Go to settings and check that the Email Protection section is visible both when logged in and logged out.
